cmd/kogito/command/install/kogito_operator.go

```diff
@@ -29,6 +29,7 @@ type installKogitoOperatorFlags struct {
 	installMgmtConsole bool
 	installAllServices bool
 	force              bool
+	channel            shared.KogitoChannelType
```

Comment:
why not using a simple string here and then parse it in `Exec` ?
Also, you would not need the `IsValid()` method

Reply:
incorporated review comments